internal/backuprestore/backup.go

```diff
@@ -142,13 +144,73 @@ func (h *BRHandler) StartOrTrackBackup(ctx context.Context, backups []*velerov1.
 	return &bt, nil
 }
 
+func getResourcesFromAnnotations(backup *velerov1.Backup) ([]GVRNamespaceName, error) {
```

Comment:
just for reference can you add func comment with the expected format (what you have in description) and "full" example input may look like?